Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ however, it has to be formatted properly to pass verification tests.
   - [`InputAction.WasCompletedThisFrame`](xref:UnityEngine.InputSystem.InputAction.WasCompletedThisFrame) returns `true` on the frame that the action stopped being in the performed phase. This allows for similar functionality to [`WasPressedThisFrame`](xref:UnityEngine.InputSystem.InputAction.WasPressedThisFrame)/[`WasReleasedThisFrame`](xref:UnityEngine.InputSystem.InputAction.WasReleasedThisFrame) when paired with [`WasPerformedThisFrame`](xref:UnityEngine.InputSystem.InputAction.WasPerformedThisFrame) except it is directly based on the interactions driving the action. For example, you can use it to distinguish between the button being released or whether it was released after being held for long enough to perform when using the Hold interaction.
 - Added Copy, Paste and Cut support for Action Maps, Actions and Bindings via context menu and key command shortcuts.
 - Added Dual Sense Edge controller to be mapped to the same layout as the Dual Sense controller
+- Added Drag and Drop support in the Input Action Asset Editor for Action Maps, Actions and Bindings.
```

Comment:
Minor but I noticed drag and drop is capitalised here but its not a named concept in the product similar to e.g. Cut, Paste, Copy where such labels exist so would make sense with just "drag and drop" or "drag-and-drop" here instead?

Reply:
fixed in [cfbd28b](https://github.com/Unity-Technologies/InputSystem/pull/1831/commits/cfbd28b411320dc946953a23f868446fe86855f2)